.env.example

```diff
@@ -3,13 +3,13 @@ ECR_REPOSITORY_NAME=cohere-embed-v3-ecr-repository
 APP_NAME=cohere-embed-v3
 IMAGE_VERSION=latest
 DOCKERFILE_NAME=Dockerfile
-PLATFORM=LINUX_ARM64 # LINUX_AMD64
+PLATFORM=LINUX_ARM64 # LINUX_AMD64, LINUX_ARM64
 PORT=8090
 CDK_DEPLOY_REGION=ap-southeast-1
 
 COHERE_API_KEY=
 COHERE_EMBED_MODEL=embed-english-v3.0
 DATA_INGESTION_API_KEY=
 
-CPU_TYPE=ONE_VCPU
-MEMORY_TYPE=TWO_GB
+CPU_TYPE=ONE_VCPU # ONE_VCPU, TWO_VCPU, FOUR_VCPU, QUARTER_VCPU, HALF_VCPU
+MEMORY_TYPE=TWO_GB # TWO_GB, FOUR_GB, EIGHT_GB, TWELVE_GB, ONE_GB, HALF_GB, SIX_GB
```

Comment:
**Suggestion:** Consider adding comments to explain the acceptable values for `CPU_TYPE` and `MEMORY_TYPE` to make it easier for users to configure these variables correctly.
> 🥈  **Medium Priority** 
 > [documentation, importance: 5]
```suggestion
CPU_TYPE=ONE_VCPU # Acceptable values: ONE_VCPU, TWO_VCPU, FOUR_VCPU, QUARTER_VCPU, HALF_VCPU
MEMORY_TYPE=TWO_GB # Acceptable values: TWO_GB, FOUR_GB, EIGHT_GB, TWELVE_GB, ONE_GB, HALF_GB, SIX_GB
```